src/components/BibleDropDown/BibleBooks.js

```diff
@@ -41,24 +57,37 @@ const useStyles = makeStyles((theme) => ({
     paddingLeft: '58px',
     width: 150,
   },
+
+  DialogTitle: {
+    fontWeight: 'bold',
+  },
+  DialogContent: {
+    paddingLeft: '5px',
+    paddingRight: '5px',
+    width: '590px',
+  },
+  displayBooks: {
+    paddingLeft: '30px',
+    marginTop: '5px',
+  },
+  displaySelectAll: {
+    paddingLeft: '20px',
+  },
+  DialogActions: {
+    paddingRight: '5%',
+  },
 }));
 
 export default function BibleBooks(props) {
   const classes = useStyles();
-  const { books } = useContext(BookContext);
```

Comment:
Since you are not using context API you can remove the surrounding context component and put this code directly in BibleDropDown.js , its not a breaking change but all that code is not necessary now. Only do it if you have time. else you can merge it as it is.